File: .skills/openclaw-skills/skills/zengjwmail/rrbdagent/list_videos.js

const RRBDClient = require('./api_client');
const config = require('./config.json');

async function listVideos() {
  console.log('============================================================');
  console.log('RRBD 视频列表查询');
  console.log('============================================================');

  const client = new RRBDClient(config);

  try {
    // 1. 登录
    console.log('\n1. 登录中...');
    console.log('   账号:', config.username);
    await client.login();
    console.log('   登录成功！');

    // 2. 查询视频列表
    console.log('\n2. 查询视频列表...');
    const videoList = await client.getVideoList();

    console.log('\n视频列表:');
    console.log('============================================================');

    if (videoList &amp;&amp; videoList.data &amp;&amp; videoList.data.length &gt; 0) {
      videoList.data.forEach((video, index) =&gt; {
        const statusText = {
          'processing': '⏳ 处理中',
          'succeed': '✅ 成功',
          'failed': '❌ 失败'
        }[video.status] || video.status;

        console.log(`\n${index + 1}. [${statusText}] ${video.title || '无标题'}`);
        console.log(`   ID: ${video.id}`);
        console.log(`   状态: ${video.status}`);
        console.log(`   创建时间: ${video.createDate}`);

        if (video.status === 'succeed' &amp;&amp; video.videoUrl) {
          console.log(`   ✨ 视频URL: ${video.videoUrl}`);
        }

        if (video.message) {
          console.log(`   消息: ${video.message}`);
        }
      });
    } else {
      console.log('   暂无视频');
    }

    console.log('\n============================================================');

  } catch (error) {
    console.error('\n❌ 错误:', error.message);
    if (error.response) {
      console.error('响应数据:', error.response.data);
    }
  }
}

listVideos();

File: .skills/openclaw-skills/skills/zengjwmail/rrbdagent/list_videos.py

import sys
import os

# 添加当前目录到 Python 路径
sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'scripts'))

from api_client import RRBDClient
import config

def main():
    print("RRBD 视频列表查询")
    print("=" * 60)
    
    # 初始化客户端
    client = RRBDClient(
        base_url=config.API_BASE_URL,
        tenant_id=config.TENANT_ID
    )
    
    # 登录
    print("\n1. 登录中...")
    print(f"   账号: {config.USERNAME}")
    
    if not client.login(config.USERNAME, config.PASSWORD):
        print("   登录失败！")
        return
    
    print("   登录成功！")
    
    # 查询视频列表
    print("\n2. 查询视频列表...")
    video_list = client.get_video_list(page_num=1, page_size=20)
    
    print("\n视频列表:")
    print("-" * 60)
    
    if video_list and video_list.get('data'):
        records = video_list['data'].get('records', []) or video_list['data']
        
        if isinstance(records, list) and len(records) &gt; 0:
            for idx, video in enumerate(records, 1):
                status_text = {
                    'processing': '⏳ 处理中',
                    'succeed': '✅ 成功',
                    'failed': '❌ 失败'
                }.get(video.get('status'), video.get('status'))
                
                print(f"\n{idx}. [{status_text}] {video.get('title', '无标题')}")
                print(f"   ID: {video.get('id')}")
                print(f"   创建时间: {video.get('createDate')}")
                
                if video.get('status') == 'succeed' and video.get('videoUrl'):
                    print(f"   ✨ 视频URL: {video.get('videoUrl')}")
        else:
            print("   暂无视频")
    else:
        print("   暂无视频")
    
    print("\n" + "=" * 60)

if __name__ == "__main__":
    main()
